CIP-????/README.md

```diff
@@ -224,24 +224,36 @@ The purpose of this CIP mainly consists of two parts. It addresses the current l
 
 ### Acceptance Criteria
 
-- [ ] A library should be build to make it easy from dAPP and wallet side to implement the proposed communication method
-- [ ] The library target should be browser to avoid the need of manual polyfills
-- [ ] Mobile testing on different devices and operating systems needs to be done with a special focus to the wallet app running in background mode
+- [x] A library should be build to make it easy from dAPP and wallet side to implement the proposed communication method
+- [x] The library target should be browser to avoid the need of manual polyfills
+- [x] Mobile testing on different devices and operating systems needs to be done with a special focus to the wallet app running in background mode
 - [ ] Potential security issues and attack vectors need to be discussed in detail
 - [ ] A full reference implementation is needed to test if the entire user flow and at the same time provide this as a how-to for developers
 
 ### Implementation Plan
 
-- [ ] Fork/Extend bugout to add webpack 5 and typescript support
-- [ ] Povide a general intermediate cardano-connect typescript library to provide 
+- [x] Fork/Extend bugout to add webpack 5 and typescript support
+- [x] Povide a general intermediate cardano-connect typescript library to provide 
     1. A check for mobile/desktop environment
     2. Depending on the environment provide interfaces for CIP-0030 / and / or CIP-?
     3. Add a full implementation of the server/client side code above to define a communication standard similar to CIP-0030 (getRewardAddresses, signData, signTx, ...)
 - [ ] Start discussions about security gaps within the proposed method with various developers and also look for research papers
 - [x] Check if the wallet app also reacts to rpc calls in background mode on Android
-- [ ] Check if the wallet app also reacts to rpc calls in background mode on iOS
+- [x] Check if the wallet app also reacts to rpc calls in background mode on iOS
 - [ ] Implement the library within an example dApp and [boost wallet](https://github.com/boost-pool/boost-wallet)
 
+### Updates
```

Comment:
These are good updates but editorially we can't allow a section for it here, since the comments would be out of flow with the standardised sections (and therefore difficult to process for readers & in "diffs")
- Some of these items would be notes for the Specification;
- notes about Implementations should be ticked boxes in the Implementation Plan above (I think they already are?)
- notes about the wording don't need to be referenced at all... we will see them from the PR's commit history... and you can also call attention to them in the PR comments to help reviewers focus on your new material 😎